src/platform/qt/ShaderSelector.h

```diff
@@ -28,17 +28,19 @@ Q_OBJECT
 	~ShaderSelector();
 
 public slots:
+	void saveSettings();
 	void refreshShaders();
 	void clear();
+	void revert();
 
 private slots:
 	void selectShader();
-	void loadShader(const QString& path);
-	void clearShader();
+	void loadShader(const QString& path, bool saveToSettings = false);
+	void clearShader(bool saveToSettings = false);
 	void buttonPressed(QAbstractButton*);
 
 signals:
-	void saved();
+	void saveSettingsRequested();
```

Comment:
Strictly speaking, just a cosmetic change, but I think it more accurately reflects the semantics of the signal.

Reply:
Sure, sounds good.